Assets/Scenes/scripts/UI/DialogueManager.cs

```diff
@@ -0,0 +1,113 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class DialogueManager : MonoBehaviour
+{
+    public Image textBox;
+    public TextMeshProUGUI text;
+
+    public string[] sentences;
+    string currentText;
+    int index;
+    bool isRunning;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        text.text = "";
+        currentText = "";
+        index = -1;
+        isRunning = false;
+        ToggleTextBox();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Escape))
```

Comment:
Not now, but later we'll have to use actions rather than Input for a more standardized key system.